frontend/src/components/InGame/helper/GamePlaying.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <div v-if="isJudge || submitted">
+    <h6 class="mt-5 text-center">Waiting for other players to submit ...</h6>
+  </div>
+  <div v-else>
+    <div class="mt-5 card-fan" :class="`fan-count-${player.deck.length}`">
+      <template v-for="(cardList, index) in [selectedCards, notSelectedCards]">
+        <div
+          v-for="card in cardList"
```

Comment:
Why not only have one `v-for` that iterates over `[...selectedCards, ...notSelectedCards]`?

Reply:
Because we need to know whether it is from selected or not selected (using the index)

Reply:
Ah, sorry, didn't see the index.